CHANGES.md

```diff
@@ -43,6 +43,7 @@
 ### Performance
 
 <!-- Changes that improve Black's performance. -->
+- Speed up the `is_fstring_start` function in `blib2to3` tokenization using generic python `startswith` function (#4541)
```

Comment:
```suggestion
- Speed up the `is_fstring_start` function in Black's tokenizer (#4541)
```
"Generic function" is more likely to confuse readers; for me at least it suggests a "generic" in the typing sense. And we don't need to go into too much detail here.